### PR TITLE
feat: add instructions for using DNS whilst developing

### PR DESCRIPTION
**Pull request by pmuir**
Signed-off-by: Pete Muir <pmuir@bleepbleep.org.uk>

### CODE REVIEW

File: content/contribute/contribute_dns.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+weight: 30
+sections_weight: 30
+draft: false
+toc: true
+---
+
+Some features and issues require DNS to be configured. The domain name `jenkins-x.rocks` is available for
+use.
+
+To add records, visit [the Cloud DNS dashboard](https://console.cloud.google.com/net-services/dns/zones/jenkins-x-rocks?project=jenkins-x-rocks&organizationId=41792434410).
```
Comment (rawlingsj) [inline content/contribute/contribute_dns.md]: FWIW we have a command to do this

```
jx create domain gke --domain jenkins-x.rocks
```
Am gonna try a live demo of this tomorrow at JW.

Comment (rawlingsj) [reply content/contribute/contribute_dns.md]: This works with external-dns BTW

Comment (pmuir) [reply content/contribute/contribute_dns.md]: Yeah, I should probably mention external dns too as people will want to debug that too. I'll ask Cai to help.

Does the `jx create domain` work even if you aren't connected to a cluster?

Comment (pmuir) [reply content/contribute/contribute_dns.md]: @cagiti how can we improve this to discuss external dns? Goal here is a contributor guide for debugging so trying to cover all paths.

**Pull request merged**